private/pkg/transport/http/httpserver/zap.go

```diff
@@ -90,3 +119,31 @@ func getRequestFields(request *http.Request) []zap.Field {
 		zap.String("path", request.RequestURI),
 	}
 }
+
+// this gets request fields that are relevant but not covered by http request log
+func getTopLevelRequestFields(request *http.Request) []zap.Field {
+	return []zap.Field{
+		zap.String("host", request.Host),
+		zap.String("path", request.RequestURI),
+	}
+}
+
+func newHTTPRequestLog(r *http.Request, status int, responseSize int, duration time.Duration) *httpRequestLog {
```

Comment:
nit: I suspect the current form is the result of following examples in the existing code, but I think we should condense parameter declaration with like types; e.g.

```go
func newHTTPRequestLog(r *http.Request, status, size int, duration time.Duration) *httpRequestLog {
// ...
}
```